core/src/main/resources/hudson/Messages.properties

```
@@ -110,6 +110,7 @@ ProxyConfiguration.TestUrlRequired=Test URL is required.
 ProxyConfiguration.MalformedTestUrl=Malformed Test URL {0}.
 ProxyConfiguration.FailedToConnectViaProxy=Failed to connect to {0}.
 ProxyConfiguration.FailedToConnect=Failed to connect to {0} (code {1}).
+ProxyConfiguration.NonTLSWarning=Please note that the connection between Jenkins and the proxy is always via an HTTP connection. Therefore, this insecure connection may expose the credentials provided.
```

maybe something like this? we shouldn't be including `please note` its verbose for no added value, the validation messages should be more direct:

```suggestion
ProxyConfiguration.NonTLSWarning=Jenkins only supports using an http connection to the proxy. The credentials may be exposed to someone on the same network.
```

---

Thanks for the suggestion, I updated the messages in b1780f7628c2d16c2eb2082787a803affb8b8c20.